### PR TITLE
ProGuard roles | Reduce apk size

### PR DESCRIPTION
**Pull request by LinX64**
This enables progaurd + shrinking resources, also reduces the size of the apk for release mode.

### CODE REVIEW

**Comment by LinX64** [comment]
[ProGuard | Recording fix session](https://trello.com/c/zyh9c5nw/1615-proguard-recording-fix-session)

File: app/src/main/java/pl/llp/aircasting/util/helpers/sensor/SessionManager.kt [hunk 1]
```diff
@@ -154,11 +154,11 @@ class SessionManager(
...
 
-    private fun registerToEventBus() {
+    fun registerToEventBus() {
         EventBus.getDefault().safeRegister(this)
     }
 
-    private fun unregisterFromEventBus() {
+    fun unregisterFromEventBus() {
         EventBus.getDefault().unregister(this)
     }
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/util/helpers/sensor/SessionManager.kt:163]: I think these methods were called from the class itself only, or are they being called from somewhere else? If no, they can be private I guess

**Review by xameYeoh — COMMENTED** [review]
Left comments

File: app/src/main/java/pl/llp/aircasting/util/helpers/sensor/SessionManager.kt (hunk 1 above)
Comment (LinX64) [reply app/src/main/java/pl/llp/aircasting/util/helpers/sensor/SessionManager.kt:163]: I need to check to see if this doesn't affect the `EventBus` with the ProGuard. but, good point! I'll look into it again!

**Review by xameYeoh — APPROVED** [review]

**Pull request merged**